src/pymmcore_widgets/_mda/_util/_labeled_slider.py

```diff
@@ -10,39 +16,34 @@ class QLabeledSlider(QtWidgets.QWidget):
     sliderReleased = QtCore.Signal()
 
     def __init__(
-        self, name: str = "", orientation=QtCore.Qt.Horizontal, *args, **kwargs
-    ):
-        # super().__init__(self, *args, **kwargs)
-        super().__init__()
+        self,
+        name: str = "",
+        orientation: QtCore.Qt.Orientation = QtCore.Qt.Orientation.Horizontal,
+        parent: QtWidgets.QWidget | None = None,
+    ) -> None:
+        super().__init__(parent)
         self.name = name
 
         self.label = QtWidgets.QLabel()
         self.label.setText(name.upper())
-        self.label.setAlignment(QtCore.Qt.AlignRight)
-        self.label.setSizePolicy(
-            QtWidgets.QSizePolicy.Fixed, QtWidgets.QSizePolicy.Fixed
-        )
+        self.label.setAlignment(QtCore.Qt.AlignmentFlag.AlignRight)
+        self.label.setSizePolicy(FIXED, FIXED)
         self.slider = QtWidgets.QSlider(orientation)
         for function in [
```

Comment:
i know it makes it much more verbose to have to define all of these methods on this class... but from a typing and IDE discovery perspective, I think it's important.  For example, there would be no typing or IDE autocompletion/awareness if you did `QLabeledSlider().value()` (though it would work at runtime of course).  I added a few of the methods that you use internally at the bottom.